src/components/molecules/NavSearchBar/NavSearchBar.tsx

```diff
@@ -1,18 +1,21 @@
 import React, { useCallback, useEffect, useState } from "react";
 
-import { VenueEvent } from "types/venues";
-
-import UserProfileModal from "components/organisms/UserProfileModal";
 import { RoomModal } from "components/templates/PartyMap/components";
+import { NavSearchBarInput } from "./NavSearchBarInput";
+
 import { useWorldUsers } from "hooks/users";
 import { useSelector } from "hooks/useSelector";
-import { Room } from "types/rooms";
-import { User } from "types/User";
+import { useProfileModalControls } from "hooks/useProfileModalControls";
+
 import { WithId } from "utils/id";
 import { currentVenueSelectorData, venueEventsSelector } from "utils/selectors";
 import { isTruthy } from "utils/types";
+
+import { Room } from "types/rooms";
+import { VenueEvent } from "types/venues";
+import { User } from "types/User";
+
 import "./NavSearchBar.scss";
-import { NavSearchBarInput } from "./NavSearchBarInput";
```

Comment:
[nit] Import sorting order, based on what is here:

- `types/*`
- `utils/*`
- `hooks/*`
- `components/templates/*`
- `components/organisms/*`
- `./*`
- `./*.scss`